pkg/ddevapp/ddevapp_test.go

```diff
@@ -235,6 +235,20 @@ var (
 			DynamicURI:                    testcommon.URIWithExpect{URI: "/", Expect: "This is a test page"},
 			FilesImageURI:                 "/sites/default/files/2022-07/Logo.png",
 		},
+		// 13: craftcms
+		{
+			Name:                          "TestPkgCraftCms",
+			SourceURL:                     "https://craftcms.com/latest-v4.zip",
+			ArchiveInternalExtractionPath: "craftcms-v4",
+			FilesTarballURL:               "",
```

Comment:
For FilesTarballURL, just upload a set of files from your dummy site, preferably a smaller set, but I like one that has files that work in a real site.

Reply:
Is this essentially Craft's source code, along with the entire vendor directory and other dependencies?

Reply:
Ok, I've packaged up files assuming they'll be imported into the public `web` directory. 
Here's the tarball: [craftcms-v4.2.3.tar.gz](https://github.com/khalwat/ddev/files/9485501/craftcms-v4.2.3.tar.gz)